docs/versioned-plugins/codecs/rubydebug-index.asciidoc

```diff
@@ -0,0 +1,14 @@
+:plugin: rubydebug
+:type: codec
+
+include::{include_path}/version-list-intro.asciidoc[]
+
+|=======================================================================
+| Version | Release Date 
+| <<v3.0.4-plugins-codecs-rubydebug,v3.0.4>> | Aug 21, 2017 (latest)
+| <<v3.0.3-plugins-codecs-rubydebug,v3.0.3>> | Aug 21, 2017
```

Comment:
this also seems easy to template, I wonder if this release date should be the one corresponding to the github tag or the rubygems release date (they'll likely be the same, but not necessarily).